src/ci/ci-rerun.js

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
+
+const { default: chalk } = require('chalk');
+const program = require('commander');
+const got = require('got');
+
+const { fatal } = require('../utils/logging');
+
+program
+  .description('Show information about CI job statuses')
+  .argument('[workflow]', 'The ID of the workflow to rerun')
+  .option('-f, --from-failed', 'Rerun workflow from failed', false)
+  .option('-e, --enable-ssh', 'Rerun workflow from failed', false)
```

Comment:
Option description got copy-pasted.

Reply:
oops, ty

Reply:
@codebytere, I actually didn't spot that one, I had spotted the description for `--enable-ssh` being the same as `--from-failed`. 😅